src/components/More.js

```diff
@@ -33,10 +34,29 @@ export class More extends React.Component {
     super(props);
     this.state = {expanded: false};
   }
+
+  // https://discuss.reactjs.org/t/how-to-calculate-jqyery-closest-in-react/1988
+  componentDidMount() {
+    let app = document.getElementById('app');
```

Comment:
The other alternative which I had considered was storing the second argument to `ReactDOM.render` in `src/dev-app/index.js` in some `window.__appDiv` variable, allowing for a single point of control.